src/frontend/src/components/Banner/Banner.jsx

```diff
@@ -19,18 +19,36 @@ const useStyles = makeStyles(theme => ({
     fontFamily: 'Roboto',
     fontWeight: 'bold',
     opacity: 0.85,
-    fontSize: '12vw',
+    fontSize: '12rem',
+    [theme.breakpoints.between('xs', 'sm')]: {
+      fontSize: '4rem',
+    },
+    [theme.breakpoints.between('md', 'lg')]: {
+      fontSize: '8rem',
+    },
+    [theme.breakpoints.up('xl')]: {
+      fontSize: '12rem',
+    },
     display: 'block',
     top: theme.spacing(20),
     left: theme.spacing(8),
+    transition: 'linear 250ms all',
   },
   stats: {
     position: 'absolute',
     color: 'white',
-    fontFamily: 'Roboto',
-    fontWeight: '400',
+    fontFamily: 'Roboto Mono',
```

Comment:
I must not have this font installed, because it's looking really different for me.  We should include some fallbacks in the font stack here, so we at least get the right family.

<img width="1364" alt="Screen Shot 2020-03-14 at 5 49 52 PM" src="https://user-images.githubusercontent.com/427398/76691010-4de21100-661c-11ea-9c77-97d9f2507e18.png">